ci/benchmark_parser.py

```diff
@@ -108,12 +108,17 @@ def recursive_parse(directory, walk_subdirs=False, name_suffix="", compute_throu
                     )
                 )
 
+                # This is a special case where PBS are blasted as vector of 512 LWE ciphertext
+                # to saturate the machine. To get the actual throughput we need to multiply by the
+                # length of the vector.
+                multiplier = 512 if "PBS_throughput" in test_name else 1
+
```

Comment:
some benchmarks have different chunk sizes though ?

Reply:
This is the only one we've set up to saturate the machine.

Reply:
What I mean is that we have other throughput stuff, and this looks like a hack more than actual solution to compute throughput for all cases we are interested in

Reply:
Indeed, this is clearly a hack. That said, I haven't found a way at criterion level to pre-process the value.

Reply:
Can we detect « chunk » in the bench name ? I think it’s the only benchmarks with chunk in the name and then parse the chunk size from there